src/containers/ProfileContainer.js

```diff
@@ -0,0 +1,170 @@
+import { connect } from 'react-redux';
+/* eslint-disable comma-dangle */
+import React from 'react';
+import propTypes from 'prop-types';
+import Cookies from 'js-cookie';
+import {
+  getProfileDetails, updateProfileDetails, getFollowers, getFollowing, updated as updatedFunc,
+} from '../actions/profileActions';
+import Profile from '../components/Profile/Profile';
+import Header from '../components/Header/Header';
+// import SessionExpiredComponent from '../common/';
+import Footer from '../components/Footer/Footer';
+import SessionExpiredComponent from '../components/common/SessionExpiredComponent';
+
+export class ProfileContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      bio: '',
+      phone: '',
+      country: '',
+      website: '',
+      email: '',
+      avatar: '',
+    };
+  }
+
+  componentDidMount() {
+    const { getFollowersFunc, getFollowingFunc } = this.props;
+    document.title = 'Profile | AuthorsHaven'; // update document title
+    const token = Cookies.get('access_token');
+    const username = Cookies.get('username');
+    this.dispatchFetchProfileData(username, token);
+    getFollowersFunc(username, token);
+    getFollowingFunc(username, token);
```

Comment:
Why is this function being called twice?

Reply:
it's not. They are two different functions, getting the followers and for getting the following or a user